src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,44 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String ladle;// ковш
```

Comment:
Use comments, where you want to explain why this or that decision was made if it's not obvious. No need to leave translates🙂